bin/yaml/libraries.yaml

```diff
@@ -2722,6 +2722,34 @@ libraries:
       - 1.16.0
       - 1.17.0
       type: github
+  kokkos:
```

Comment:
we'll need to find a way to install this in a different directory to the other kokkos; as these currently will land in the same directory. I'll clone locally and see if I can get this working.